src/admin/scss/custom/websites.scss

```diff
@@ -0,0 +1,196 @@
+.website-edition-form {
+	//font-size: 20px;
+
+	label {
+		text-transform: uppercase;
+		color: #777;
+		font-size: 1.2rem;
+		border-bottom: 1px solid #dddddd;
+		margin-bottom: 1rem;
+
+		.button {
+			margin: -.4rem .4rem;
+			float: right;
+		}
+	}
+
+	input[type='text'], select {
+		//border: 1px solid #597e97;
+		//border-radius: 5px;
+		font-family: 'Roboto', sans-serif;
+		font-size: .9rem;
+		margin-bottom: 1rem;
+		margin-left: .5rem;
+	}
+
+	input[type='text'] {
+		padding: .5rem;
+	}
+
+	fieldset {
+		margin-bottom: 1rem;
+	}
+
+	.name-section {
+		margin-bottom: 1rem;
+	}
+
+	.fa {
+		&:before {
+			color: #FF0000;
+			margin-left: .8rem;
+			cursor: pointer;
+		}
+	}
+
+	.website-host-list, .website-editor-list {
+		margin-bottom: 5px;
+		padding: 5px;
+		font-size: 1rem;
+		list-style: none;
+	}
+
+	.website-parser-class, .website-templates >span {
+		margin-left: 1.25rem;
+	}
+}
+
+.website-edit-form {
+	.row {
+		margin-bottom: 1rem;
+	}
+}
+
+.website-hosts-list, .website-editors-list {
+	list-style: none;
+	margin-left: 0;
+}
+
+.website-hosts-list-item, .website-editors-list-item {
+	margin-right: 5px;
+	margin-bottom: 2px;
+	padding: 4px;
+	font-size: 1rem;
+	display: inline-block;
+	color: #fff;
+	font-weight: 600;
+
+	.fa {
+		&:before {
+			color: #fff;
+			margin-left: .9rem;
+			cursor: pointer;
+		}
+	}
+}
+
+.website-hosts-list-item {
+	background-color: #1a7bf3;
+}
+
+.website-editors-list-item {
+	background-color: #3adb76;
+}
+
+.websites-group-holder {
+	.websitelist-table-heading, .websitelist-table-row {
+		background: #CCCBCB;
+		padding: 0 .9375rem;
+	}
+
+	.websitelist-table-heading {
+		font-weight: bold;
+		font-style: italic;
+		text-transform: uppercase;
+	}
+}
+
+.additional-settings-section {
+	.additional-content {
+		max-height: 0;
+		overflow-y: hidden;
+		margin-top: 10px;
+		-webkit-transition: max-height 0.3s ease-in-out;
+		-moz-transition: max-height 0.3s ease-in-out;
+		-o-transition: max-height 0.3s ease-in-out;
+		transition: max-height 0.3s ease-in-out;
+
+
+		.additional-content-section {
+			border: 1px solid #cacaca;
+			padding: 0.9375rem 0.9375rem;
```

Comment:
make run/lint.. Coz Sass-linter doesn't pass 0.9375rem. Maybe it exists other sass-issues